src/sensor.cc

```diff
@@ -0,0 +1,79 @@
+#include "sensor.h"
+#include "vector3.h"
+
+namespace node_sfml {
+namespace gen {
+extern void InitSensorType();
+}  // namespace gen
+
+namespace sensor {
+
+using std::map;
+using std::string;
+using v8::Local;
+using v8::MaybeLocal;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+NAN_MODULE_INIT(Init) {
+  gen::InitSensorType();
+
+  Nan::SetMethod(target, "sensorIsAvailable", IsAvailable);
+  Nan::SetMethod(target, "sensorSetEnabled", SetEnabled);
+  Nan::SetMethod(target, "sensorGetValue", GetValue);
+
+  Local<Object> type = Nan::New<Object>();
+  for (auto it = gen::sensor_type_atoi.begin(); it != gen::sensor_type_atoi.end();
+       it++) {
+    Nan::Set(type,
+             Nan::New<String>(it->first).ToLocalChecked(),
+             Nan::New(it->second));
+  }
+
+  Nan::Set(target, Nan::New<String>("SensorType").ToLocalChecked(), type);
+}
+
+NAN_METHOD(IsAvailable) {
+  if (info.Length() != 1) {
+    Nan::ThrowError("Invalid arguments count.");
+    return;
+  }
+  
+  auto sensorType = static_cast<sf::Sensor::Type>(Nan::To<sf::Uint32>(info[0].As<v8::Uint32>()).FromJust());
```

Comment:
We should pass the CPPLINT:

```bash
$ npm run cpplint
```

A rule here is that max line length is 80. We may run `$ npm run clang-format` to format CPP files.

Reply:
Did run all of that, now it's passing without errors :)